### ISSUE TITLE
Can't mointor multi gpu devices

### ISSUE THREAD

**Issue by reasonW** [fossfreedom/indicator-sysmonitor#107]
Hi, this monitor can only fetch the memory/temperature of the GPU DEVICE '0'. It doesn't work on a multi GPU machine.

**Comment by fossfreedom** [comment]
Fair enough. Since I don't have multi gpu capability I will love to see a PR providing such a capability.
